Sources/Prometheus/MetricTypes/Gauge.swift

```diff
@@ -136,8 +141,8 @@ public class Gauge<NumType: Numeric, Labels: MetricLabels>: Metric, PrometheusHa
     ///     - labels: Labels to attach to the value
     ///
     public func dec(_ labels: Labels? = nil, _ done: @escaping (NumType) -> Void = { _ in }) {
-        self.dec(1, labels) {
-            done($0)
+        self.dec(1, labels) { value in
+            done(value)
```

Comment:
would you mind changing this to
```swift
self.dec(1, labels, done)
```
instead? Not sure why I did not do that in the first place